PowerScout/AppDelegate.swift

```diff
@@ -16,6 +16,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
+        
+        // TODO: REMOVE WHEN NOT USING MOCK
+//        MatchStore.useMock = true
```

Comment:
When testing the data transfer, Uncomment this to populate the match store with mock data. 
(remember to uncomment when done testing)